### PR TITLE
Add lesson topic info/guidance

### PR DESCRIPTION
**Pull request by tobyhodges**
Introducing some information about lesson repository topics for Incubator repos, and linking to the sheet of all topics currently in use across the Incubator.

Not sure if this is the best place to be adding this information, so please advise if you think it's a better fit somewhere else.

### CODE REVIEW

File: 07-lesson-life-cycle.Rmd [hunk 1]
```diff
@@ -73,15 +73,18 @@ support.
 
 Before you start developing a new lesson, check
 to see if there are already people working on creating
-a lesson for this topic. The [Carpentries Incubator][incubator] is where our community comes
-together to talk about lesson ideas and find 
-collaborators. You can check existing [issues][issues] 
+a lesson for this topic.
+The [Carpentries Incubator][incubator] is where our community
+develops new curriculum together.
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:78]: curriculum --> curricula

File: 07-lesson-life-cycle.Rmd [hunk 2]
```diff
@@ -73,15 +73,18 @@ support.
...
+New lessons are suggested for development and added to Incubator via the [Proposals][proposals] repository,
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:79]: "the Incubator"

File: 07-lesson-life-cycle.Rmd [hunk 3]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
-and provide pedagogical guidance. For lessons on the 
-Official Track, authors will meet regularly with 
+and provide pedagogical guidance. For lessons on the
+Official Track, authors will meet regularly with
 a member of the Curriculum Team to work through the
 lesson drafting process.
 
+Lessons under community development in the Carpentries Incubator are
+[listed on The Carpentries website][community-lessons],
+based on topics added to the lesson repository.
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:150]: topics --> "topic tags"?

Comment (ErinBecker) [reply 07-lesson-life-cycle.Rmd:150]: I know these metadata elements are referred to by GitHub just as "topics", but I think that usage is a bit outside of normal vernacular English usage and can be confusing. I'm proposing that we refer to GitHub "topics" as "topic tags" and/or explicitly state somewhere that these are metadata elements and link to GitHub documentation about "topics".

File: 07-lesson-life-cycle.Rmd [hunk 4]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+Some topics are essential, while others are more flexible.
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:151]: topics --> "topic tags"

File: 07-lesson-life-cycle.Rmd [hunk 5]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+See the table below for guidance about the types and number of topics each
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:152]: topics --> "topic tags"

File: 07-lesson-life-cycle.Rmd [hunk 6]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+|----------|----------|--------|------|
+| Lesson* | `lesson` | 1 | Must be `lesson` to be listed on the Community Developed Lessons page. |
+| Location* | `carpentries-incubator` | 1 | `carpentries-incubator` or `carpentrieslab`. |
+| Language* | `español` | >0 | The language(s) the lesson is available in. |
+| Stage* | `alpha` | 1 | The current development stage for the lesson. Read on for more info. |
+| Domain | `microbial-ecology` | 1-2 | The high-level domain(s) of the lesson, for a general categorization. |
+| Tools | `python` | 1-3 | The main tool(s) taught in the lesson. |
+| Skills | `taxonomic-classification` | 1-3 | The main skill(s) taught in the lesson. |
+
+*Categories marked with an asterisk (\*) are essential.*
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:165]: essential --> "required in order for your lesson to appear and be appropriately sorted on the Community Developed Lessons page."

File: 07-lesson-life-cycle.Rmd [hunk 7]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+
+The Curriculum Team will support you in setting topics for your lesson.
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:167]: setting topics --> "setting appropriate topic tags"

File: 07-lesson-life-cycle.Rmd [hunk 8]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+To help ensure consistency across our lesson repositories,
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:168]: "our lesson repositories" - @tobyhodges could you please do a sweep through this file (and other places where you are updating documentation) to make sure we're not using "our" or "The Carpentries" when referring to lessons in the Incubator? Here, for example, we could say "to help ensure consistency across all lesson repositories developed by The Carpentries community".

File: 07-lesson-life-cycle.Rmd [hunk 9]
```diff
@@ -132,16 +135,42 @@ as high-quality resources.
...
+please refer to [this listing][incubator-topics] of topics currently
+in use in The Carpentries Incubator,
+and re-use these values where appropriate,
+creating new topics where no pre-existing label exists for your lesson.
```
Comment (ErinBecker) [inline 07-lesson-life-cycle.Rmd:172]: "creating new topic tags"

Comment (ErinBecker) [reply 07-lesson-life-cycle.Rmd:172]: Do we want people to add things directly to the spreadsheet?

**Review by ErinBecker — CHANGES_REQUESTED** [review]
@tobyhodges - looking good with a few suggested minor changes around language. Please take a look at my comments and see what you think.

**Comment by tobyhodges** [comment]
I had been using "topic tags" in my first draft but switched to "topics" when I considered that Git & GitHub already use the term "tags" to mean something else. In particular, GitHub uses it for releases and I worried that this might case more confusion. We face a similar problem with "topic labels," since labels are used on issues...

I do agree that "topic tags" reads better, though: it's clearer that we're talking about applying metadata to the lesson, rather than influencing the actual content. I'll make the changes and we can revisit if I or anyone else can think of a more distinct/appropriate term.

File: 07-lesson-life-cycle.Rmd (hunk 9 above)
Comment (tobyhodges) [reply 07-lesson-life-cycle.Rmd:172]: It's currently set to allow anyone visiting to comment on the sheets. I'd prefer to maintain the document myself. 

In the long run I'm hoping the Infrastructure team can help me with keeping the list up-to-date, but for now I'll be updating it via regular sweeps of the lesson repositories.

**Comment by tobyhodges** [comment]
Now I've finished updating, it has occurred to me that this material might be better placed in a different location. Information about working with and in The Carpentries Incubator might be more appropriate in the Technological Introductions section (or a new section on its own)? A description of adding topic tags doesn't feel like part of the discussion of lesson life cycle, adn we'll almost certainly end up adding more material about the Incubator (e.g. details of curriculum community activities, etc). What do you think, @ErinBecker?

**Comment by tobyhodges** [comment]
Based on offline discussion with @ErinBecker I'm going to move this to begin a new appendix, which will contain documentation for the Carpentries Incubator. Closing this PR now. I'll open a new one with the content in a different location.

**Pull request closed without merging**